tests/e2e/upgrade_test.go

```diff
@@ -282,12 +305,17 @@ var _ = Describe("Upgrade", Label(tests.LabelUpgrade, tests.LabelNoOpenshift), O
 				return err
 			}
 
-			var count int
+			var count, nonUpgradeCount int
+			By("checking cluster events for manager")
```

Comment:
Hi @jsilvela ,  the message `By("checking cluster events for manager")` is inside the `retry.OnError`. It may be printed multiple times. Can it be moved out of the `retry.OnError`?

Reply:
In my opinion, no, but not a strong feeling.